src/components/pages/singleArticle.jsx

```diff
@@ -8,17 +8,36 @@ const SingleArticle = () => {
 	const { article_id } = useParams();
   const [isLoading, setIsLoading] = useState(false); 
   const [date, setDate] = useState("");
+  const [likes, setLikes] = useState(0)
 
 	useEffect(() => {
     setIsLoading(true)
 		getArticleById(article_id).then((data) => {
 			setArticle(data);
       setIsLoading(false)
       setDate(data.created_at)
+      setLikes(data.votes)
      
 
 		});
 	}, []);
+
+  const handleClick = (article_id) => {
+   
+    setArticle((currArticle) => {
+      const updatedArticle = {...currArticle, votes: currArticle.votes + 1} 
+    setLikes(likes +1)
+    return updatedArticle})
+    
+    
+     addLikeToArticle(article_id).catch((err) => {
```

Comment:
we also want to decrement the likes back to the original number if the request fails, and then let the user know what has happened